src/Constants/SocketEvents.js

```diff
@@ -13,6 +13,8 @@ const SOCKET_EVENTS = {
   USER_KICKED: 'user-kicked',
   UPDATE_VIDEO_STATUS: 'update-video-status',
   VIDEO_STATUS_UPDATED: 'video-status-updated',
+  JUMP_VIDEO: 'jump-video',
```

Comment:
isim olarak `JUMP_VIDEO` başka bir videoya atlamak olarak anlaşılıyor fakat burada yaptığımız iş videonun saniyesini değiştirmek bundan dolayı isim daha anlaşılır yapılabilir.

Reply:
aklıma bir şey gelmediği için bu şekilde yaptım. Önerin nedir?